question10/main.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+// loading img 
+const wrapper = document.getElementById('wrapper');
+const loadImg = document.createElement('img');
+const showLoadImg = () => {
+    loadImg.src = "loading-circle.gif";
+    wrapper.appendChild(loadImg);
+} 
+
+const contentsList = [{to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, 
+                      {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}]
+
+
+const promise = new Promise((resolve) => {
+    setTimeout(() =>{
+      resolve(contentsList);
+    }, 3000)
+})
+
+async function asyncFunc() {
+    showLoadImg();
+    try {
+        const result = await promise;
+        createList(result);
+    }
```

Comment:
IMO

I've been told by Mr.Morita the following

Do not execute "createList" in the try.

This is because you can’t judge
whether the error is from "promise" or "createList".

Reply:
Thanks for the review and I was agree with you! 
My new idea is to remove the "createElement" from the "try"  and connect it with “then" after the async func(). Actually I do not have the confidence about this so I would like to hear the your opinion..
Latest commit : https://github.com/csakyo/homework/pull/22/commits/cd20136eb81e7ad36cd08a75c7fbcd16c0e81805

Reply:
The idea is correct.
Async await is recommended, 
so I think it would be better to use that.

Reply:
Thank you for the review!
I changed it to async/await. Is this what you mean?Please confirm.
Latest commit : https://github.com/csakyo/homework/pull/22/commits/7c273d2713e1d4c05bae2f55e6e9daefbb5141e8